stories/assets/scss/_breakpoints.scss

```diff
@@ -0,0 +1,20 @@
+@mixin devicebreak($point) {
+    @if $point==small {
+        /* small design */
+        @media (min-width: 768px) {
+            @content;
+        }
+    }
+    @else if $point==medium {
+        /* medium design */
+        @media (min-width: 1400px) {
+            @content;
+        }
+    }
+    @else if $point==large {
+        /* large design */
+        @media (min-width: 1441px) {
+            @content;
+        }
+    }
+}
```

Comment:
@harishkumar-srijan these should be the breakpoints
/* Mobiles only */
@media screen and (max-width: 47.9375em) {}

/* Above Mobiles */
@media screen and (min-width: 48em) {}

/* Tablets only */
@media screen and (min-width: 48em) and (max-width: 89.9375em) {}

/* Large Desktops */
@media screen and (min-width: 90em) {}